packages/breadboard-ui/src/events/events.ts

```diff
@@ -75,6 +76,16 @@ export class InputEnterEvent extends Event {
   }
 }
 
+export class InputErrorEvent extends Event {
+	static eventName = ErrorNames.INPUT_ERROR;
```

Comment:
nit: I wonder if we need to have this indirection? If the name of the event is set here we can refer to `InputErrorEvent.eventName` wherever we need to.